ceph.spec.in

```diff
@@ -61,6 +78,11 @@ Requires: systemd
 %if 0%{with cephfs_java}
 BuildRequires: sharutils
 %endif
+%if 0%{with selinux}
+BuildRequires:	checkpolicy
+BuildRequires:	selinux-policy-devel
+BuildRequires:	/usr/share/selinux/devel/policyhelp
```

Comment:
@tchaikov: The last time I checked, the.spec file built fine on f22 (it was a few commits back but it already contained this chunk). Either way, I scheduled a koji build on top of this branch to confirm:

http://koji.fedoraproject.org/koji/taskinfo?taskID=10598873